src/tagmanager/tm_parser.c

```diff
@@ -1080,6 +1080,18 @@ static TMParserMapGroup group_BATCH[] = {
 	{N_("Variables"), TM_ICON_VAR, tm_tag_variable_t},
 };
 
+static TMParserMapEntry map_AUTOIT[] = {
+	{'f', tm_tag_function_t},
+	{'r', tm_tag_other_t},
+	{'g', tm_tag_undef_t},
+	{'l', tm_tag_undef_t},
```

Comment:
Don't we want to see at least global variables?  And even local ones would be nice, esp. as IIUC they are now not shown but can still be used for autocompletion (@techee?)

Reply:
Yes, that would be nice I think (I didn't mention this because I thought they were disabled for some language-specific reason).

If you decide to enable local variables, please map them to `tm_tag_local_var_t` - this mapping is a bit special as it doesn't make the local variables appear in the sidebar (which is a bit too verbose with them) and only shows them in the autocompletion popup.

Reply:
Plus we want to have all the enabled tags covered by unit tests - I haven't checked if the provided unit test contains local and global tags.

Reply:
I just wanted to add minimal guaranteed working AutoIt support and don't know all the ins and outs of how Geany works.
And didn't want to bloat the list of symbols (maybe in the future if anyone wants to).

I.e. `tm_tag_variable_t` for global variables and `tm_tag_local_var_t` for local?

Reply:
I'd suggest that we address this in a separate PR (@b4n what do you think?). This PR works well as it is and I can prepare a PR for the global and local variables afterwards - it should be trivial.

Reply:
@techee sure, that'd be fine as well (although it'd probably get forgotten for a decade or two until somebody notices it from scratch again)